package.json

```diff
@@ -1450,11 +1452,32 @@
             "Put the viewer tab in a new group below the current one"
           ]
         },
+        "latex-workshop.view.pdf.tab.viewColumn": {
+          "scope": "window",
+          "type": "string",
+          "default": "Beside",
+          "markdownDescription": "The view column in which to open the tab viewer, if treating it as a custom editor.",
+          "enum": [
+            "Active",
+            "Beside",
+            "One",
+            "Two",
+            "Three"
+          ],
+          "enumDescriptions": [
+            "Use the currently active editor group",
+            "Use the column next to the current editor",
+            "Use the first editor column",
+            "Use the second editor column",
+            "Use the third editor column"
+          ]
+        },
         "latex-workshop.view.pdf.tab.openDelay": {
           "scope": "window",
           "type": "number",
           "default": 1000,
-          "markdownDescription": "Defines the delay in milliseconds to wait for a tab opening. Please increase the value if you encounter a focus issue after opening a tab."
+          "markdownDescription": "Defines the delay in milliseconds to wait for a tab opening. Please increase the value if you encounter a focus issue after opening a tab.",
+          "markdownDeprecationMessage": "Deprecated, does not have any effect."
```

Comment:
Deprecated configs should be changed to what we did for e.g. `intellisense.atSuggestionJSON.replace`